tests/execution-time-limit/00-background/suite.rc

```diff
@@ -3,23 +3,20 @@
    abort if any task fails = True
    [[events]]
        abort on inactivity = True
+       abort on stalled = True
```

Comment:
Couldn't it hide another bug? I mean, if there was a bug causing the suite to get stalled... I thought we only wanted to test inactivity here.

Reply:
It should be OK, because the main tests are:
* Whether the job file has an execution time limit header.
* Whether the job is terminated with the XCPU/ERR signal.

Reply:
(Whether the suite runs well or not is not the main focus of this test.)